packages/sit-onyx/src/components/OnyxDataGrid/features/selection/selection.scss

```diff
@@ -0,0 +1,22 @@
+@use "../../../../styles/mixins/layers.scss";
+
+@include layers.component {
+  tr:has(.onyx-data-grid-selection-cell__checkbox--checked) td {
+    background-color: var(--onyx-color-base-primary-100);
+  }
+
+  .onyx-data-grid-selection-cell {
+    box-sizing: content-box;
+    width: 2.5rem;
+    padding: 0;
+
+    &--hover &__checkbox {
+      visibility: hidden;
+    }
+
+    tr:hover &__checkbox,
```

Comment:
Shouldn't it also be possible to select via keyboard? So it should be visible on focus too